src/main/java/games/strategy/triplea/ui/menubar/ViewMenu.java

```diff
@@ -279,14 +279,8 @@ private void addShowMapDetails(final JMenu menuGame) {
         return;
       }
       TileImageFactory.setShowReliefImages(showMapDetails.isSelected());
-      final Thread t = new Thread("Triplea : Show map details thread") {
-        @Override
-        public void run() {
-          yield();
```

Comment:
This call to Thread#yield has been removed.
From the javadocs:

> A hint to the scheduler that the current thread is willing to yield its current use of a processor. The scheduler is free to ignore this hint.
> 
> Yield is a heuristic attempt to improve relative progression between threads that would otherwise over-utilise a CPU. Its use should be combined with detailed profiling and benchmarking to ensure that it actually has the desired effect.
> 
> It is rarely appropriate to use this method. It may be useful for debugging or testing purposes, where it may help to reproduce bugs due to race conditions. It may also be useful when designing concurrency control constructs such as the ones in the java.util.concurrent.locks package.

I'm conviced that no in-depth profiling has been done...

Reply:
Agreed about the profiling conclusion.  My guess is it was a hack around a race condition someone discovered.

This instance was introduced in c75d9a8, and the commit message only says, "allow toggling of the map details."